taos-optin/src/lib.rs

```diff
@@ -426,7 +450,19 @@ impl taos_query::TBuilder for TaosBuilder {
         if let Some(pass) = dsn.password.as_deref() {
             auth.pass.replace(CString::new(pass).unwrap());
         }
+
         let params = &dsn.params;
+        if let Some(totp) = params.get("totp_code") {
```

Comment:
Better support both totp_code and totpCode (camelCase), or just use consist camelCase style